beacon-chain/state/state-native/hasher.go

```diff
@@ -236,5 +238,30 @@ func ComputeFieldRootsWithHasher(ctx context.Context, state *BeaconState) ([][]b
 		fieldRoots[nativetypes.LatestExecutionPayloadHeader.RealPosition()] = executionPayloadRoot[:]
 	}
 
+	if state.version == version.Capella {
+		// Execution payload root.
+		executionPayloadRoot, err := state.latestExecutionPayloadHeaderCapella.HashTreeRoot()
+		if err != nil {
+			return nil, err
+		}
+		fieldRoots[nativetypes.LatestExecutionPayloadHeaderCapella.RealPosition()] = executionPayloadRoot[:]
+
+		// Withdrawal queue root.
+		withdrawalQueueRoot := []byte("stub")
+		fieldRoots[nativetypes.WithdrawalQueue.RealPosition()] = withdrawalQueueRoot
+
+		// Next withdrawal index root.
+		nextWithdrawalIndexBuf := make([]byte, 8)
+		binary.LittleEndian.PutUint64(nextWithdrawalIndexBuf, state.nextWithdrawalIndex)
+		nextWithdrawalIndexRoot := bytesutil.ToBytes32(nextWithdrawalIndexBuf)
```

Comment:
Why the double copy here: can't we just put the 8 bytes directly in the 32 bytes buffer?